src/App.jsx

```diff
@@ -1,34 +1,122 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getTodos } from './api/api';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
-class App extends React.Component {
-  state = {
-    todos: [],
-    selectedUserId: 0,
+const App = () => {
+  const [todos, setTodos] = useState([]);
+  const [filteredTodos, setFilteredTodos] = useState([]);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [text, addText] = useState('');
+  const [status, setStatus] = useState('All');
+
+  useEffect(() => {
+    getTodos()
+      .then((list) => {
+        const listFromServer
+          = list.data.filter(item => item.title && item.id && item.userId);
+
+        setTodos(listFromServer);
+        setFilteredTodos(listFromServer);
+      });
+  }, []);
+
+  const selectUser = (event) => {
+    setSelectedUserId(event.target.textContent);
   };
```

Comment:
```suggestion
  const selectUser = ({ target }) => {
    setSelectedUserId(target.textContent);
  };
```
why do you use textContent? value should work too